src/security/Handshake.cc

```diff
@@ -550,23 +551,26 @@ Security::HandshakeParser::ParseCertificate(const SBuf &raw, Security::CertPoint
     pCert = nullptr;
     debugs(83, 2, "TLS parsing is not supported without OpenSSL. " << raw);
 #endif
+    return pCert;
```

Comment:
Please assert that the certificate pointer is not nil before returning that pointer. It may help Coverity and humans to confirm the guarantees promised by the method description.